blog/_posts/2017-08-14-with-love-team-bundledore-india.md

```diff
@@ -0,0 +1,77 @@
+title: With love, Team Bundledore, India !
+ layout: post
+ created_at: Saturday August 14 2017
+ permalink: blog/2017-08-14-bundledore-with-love-team-Bundledore-India
+ current: blog
+ author: Bundledore
+ twitter: TeamBundledore
+ categories:student-posts
+ 
+ 
+ ### Hello, people! 
+ A very Happy Independence Day to all my fellow Indians here :) We are Team Bundledore, and our project is [AFDC League Management System](https://leagues.afdc.com/) 
+ 
+ The last one and a half-month has been crazy hectic with all the learning, connecting and mentoring. We have truly never enjoyed an internship so much before :)
+ 
+ We were sort of a 'Wild-Card' entry, our selection was confirmed in the middle of May ! It feels nice to call ourselves that :P 
+ We are die-hard Potterheads, hence the name Bundledore(Bundle+ Dumbledore). 
+ 
+ ![image](/img/blog/2017/bundledore.jpg)
+<div align="center" class="image-credits">Our Logo !</div>
+
+### About us:
+ We are both part of an extremely active [FOSS Club](http://foss.amrita.ac.in/) at our college. That's where we met. Didi(that's the term we use to refer to our seniors) was a sophomore and I was a fresher. We started our journey there, everyday gathering at 4:30 PM, to help each other learn new technologies, one of them being Rails.
```

Comment:
Please use the names instead of I or didi. It would be kind of confusing for readers to know who is referring whom over here. :)